lib/pbench/test/unit/agent/task/test_copy_result_tb.py

```diff
@@ -91,9 +91,9 @@ def request_callback(request: requests.Request):
             agent_logger,
         )
         if access is None:
-            crt.copy_result_tb("token")
+            res = crt.copy_result_tb("token")
         else:
-            crt.copy_result_tb("token", access)
+            res = crt.copy_result_tb("token", access)
         # If we got this far without an exception, then the test passes.
```

Comment:
This code comment is no longer accurate -- it should be replaced with some sort of assertion on `res`.

Ditto line 138 (et al.?).